test/.eslintrc

```diff
@@ -1,24 +1,13 @@
 {
-	"boss": true,
-	"curly": true,
-	"eqeqeq": true,
-	"eqnull": true,
-	"expr": true,
-	"immed": true,
-	"noarg": true,
-	"quotmark": "double",
-	"undef": true,
-	"unused": true,
-
-	"es3": true,
-
-	"evil": true,
-	"sub": true,
-
-	"browser": true,
-	"devel": true,
-	"wsh": true,
-
+	"env": {
+		"browser": true
```

Comment:
Just to note: here this `env` is fine, lots of tests assume the browser environment. It's just in `src/.eslintrc` where it's important to not specify it.